pkg/services/sqlstore/migrations/user_auth_oauth_mig.go

```diff
@@ -0,0 +1,25 @@
+package migrations
+
+import . "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+
+func addUserAuthOAuthMigrations(mg *Migrator) {
+	userAuthV2 := Table{Name: "user_auth"}
+
+	mg.AddMigration("Add OAuth access token to user_auth", NewAddColumnMigration(userAuthV2, &Column{
```

Comment:
I think it's ok to add those fields to the user_auth table.

But could you please move these migrations to `user_auth_mig.go`?